### ISSUE TITLE
Failed validating stash account address when monitoring validator_nodes

### ISSUE THREAD

**Issue by ivanceras** [SimplyVC/panic_polkadot#8]
Monitoring nodes as full_node has no problem, however monitoring them as validator node will give me the error:

```log
Failed validating stash account address 5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY
```

Here is the config for alice_validator_node

```ini
[node_0]
node_name = alice_validator
node_ws_url = ws://127.0.0.1:9944
node_is_validator = True
is_archive_node = False
monitor_node = True
use_as_data_source = True
; Alice stash account
stash_account_address = 5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY
;stash_account_address = 
```

where stash_account_address is configured to use alice_stash account address.

Further debugging and putting `print` in some parts of the code I can see that the ` http://localhost:3000/api/query/staking/stakers` api from polkadot is returning 404.

```log
Current working directory set to /home/user/panic_polkadot
Enabled alerting channels (general): ConsoleChannel, LogChannel, TelegramChannel, MongoChannel
Enabled alerting channels (periodic alive reminder): ConsoleChannel, LogChannel, TelegramChannel, MongoChannel
Trying to retrieve data from the API of alice_validator
Getting data..
requesting http://localhost:3000/api/rpc/system/chain
get_ret: <Response [200]>
{'result': 'Local Testnet'}
Success.
stash account is: [5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY]
Getting stakers..
{'websocket': 'ws://127.0.0.1:9944', 'account_address': '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY'}
Getting data..
requesting http://localhost:3000/api/query/staking/stakers
get_ret: <Response [404]>
Exception: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Failed validating stash account address 5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY
```

**Comment by dillu24** [comment]
Hi @ivanceras 

Thanks for opening this issue. This error is being raised because PANIC needs to still be updated to be compatible with the new version of the [polkadot-js/api](https://polkadot.js.org/api/api/)

That being said, the team will be looking into these issues soon. In the meantime please use this config 

`[node_0]
node_name = alice_validator
node_ws_url = ws://127.0.0.1:9944
node_is_validator = False
is_archive_node = False
monitor_node = True
use_as_data_source = True
; Alice stash account
stash_account_address =
;stash_account_address = `

In other words, for now please monitor all validators as full nodes.

**Comment by dillu24** [comment]
Issue solved in GH-9
